frontend/pages/hosts/ManageHostsPage/components/RunScriptBatchModal/RunScriptBatchModal.tsx

```diff
@@ -173,14 +306,51 @@ const RunScriptBatchModal = ({
       >
         <>
           {renderModalContent()}
-          <div className="modal-cta-wrap">
-            <Button disabled={isUpdating} onClick={onCancel}>
-              Done
-            </Button>
-          </div>
+          {!selectedScript && !scriptForDetails && (
+            <div className="modal-cta-wrap">
+              <Button disabled={isUpdating} onClick={onCancel}>
+                Done
+              </Button>
+            </div>
+          )}
+          {selectedScript && (
+            <div className="modal-cta-wrap">
+              <TooltipWrapper
+                tipContent="Enter a date and time to schedule this script."
+                underline={false}
+                position="top"
+                disableTooltip={formValidation.isValid}
+                showArrow
+              >
+                <Button
+                  disabled={isUpdating || !formValidation.isValid}
+                  onClick={() => onRunScriptBatch(selectedScript)}
+                  isLoading={isUpdating}
+                >
+                  Run
+                </Button>
+              </TooltipWrapper>
+              <Button
```

Comment:
The Figma just has a "Cancel" button that should behave like this, unless I missed something